coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
if the JSONEncoder has an object you can simply make a normal method and be fine. Or we could patch the JSONEncoder to do the getattr on its own.